api/src/pcapi/routes/backoffice_v3/fraud.py

```diff
@@ -0,0 +1,159 @@
+from operator import attrgetter
+
+from flask import flash
+from flask import redirect
+from flask import render_template
+from flask import url_for
+from flask_login import current_user
+from flask_sqlalchemy import BaseQuery
+import sqlalchemy as sa
+
+import pcapi.core.history.api as history_api
+import pcapi.core.history.models as history_models
+import pcapi.core.permissions.models as perm_models
+from pcapi.core.users.api import suspend_account
+import pcapi.core.users.constants as users_constants
+import pcapi.core.users.models as users_models
+
+from . import utils
+from .forms import fraud as forms
+
+
+fraud_blueprint = utils.child_backoffice_blueprint(
+    "fraud",
+    __name__,
+    url_prefix="/fraud",
+    permission=perm_models.Permissions.FRAUD_ACTIONS,
+)
+
+
+def render_domain_names_list(form: forms.BlacklistDomainNameForm | None = None) -> utils.BackofficeResponse:
+    if not form:
+        form = forms.PrepareBlacklistDomainNameForm()
+
+    history = (
+        history_models.ActionHistory.query.filter_by(actionType=history_models.ActionType.BLACKLIST_DOMAIN_NAME)
+        .order_by(history_models.ActionHistory.actionDate.desc())
+        .limit(50)
+        .options(
+            sa.orm.joinedload(history_models.ActionHistory.authorUser).load_only(
+                users_models.User.id, users_models.User.firstName, users_models.User.lastName
```

Comment:
```suggestion
                users_models.User.id, users_models.User.firstName, users_models.User.lastName, users_models.User.email
```
car `full_name` peut afficher `email` (@Vencespass me l'a soufflé sur une autre PR)